src/components/landlord-dashboard/TabContent.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import { TabsContent } from '@/components/ui/tabs';
+import { Progress } from '@/components/ui/piechart';
+import { Card, CardHeader, CardContent } from '@/components/ui/card';
+import { Key } from 'react';
+import { ICardContentProps } from '@/types/commons';
+import { Button } from '../ui/button';
+import Link from 'next/link';
+
+interface ILandlordDashboardTabContentProps {
+  tabType: string;
+  startDate: string;
+  endDate: string;
+  progressPercent: number;
+  cardContents: ICardContentProps[];
+  shareHouseId: string;
+}
+
+const LandlordDashboardTabContent = ({
+  tabType,
+  startDate,
+  endDate,
+  progressPercent,
+  cardContents,
+  shareHouseId,
+}: ILandlordDashboardTabContentProps) => {
+  return (
+    <TabsContent value={tabType}>
+      <div className="flex justify-center">
+        <div className="w-32 sm:w-48">
+          <Progress progressPercent={progressPercent} />
+        </div>
+      </div>
+      <Card className="mt-6">
+        <CardHeader
+          startDate={startDate}
+          endDate={endDate}
+          title={'Task assignment'}
+        />
+        {cardContents.map(
+          (
+            content: {
+              category: string | null;
+              tenant: string;
+              isComplete: boolean;
+              taskNum: number;
+              completedTaskNum: number;
+            },
+            index: Key,
+          ) => {
+            if (content.category === null) {
+              return (
+                <div
+                  key={index}
+                  className="flex items-center justify-center flex-col w-full py-6"
+                >
+                  <div className="pb-4">No tenants</div>
+                  <Link href={`/sharehouses/${shareHouseId}/edit?tab=Tenants`}>
+                    <Button>Add tenant</Button>
+                  </Link>
+                </div>
+              );
+            }
+            return (
+              <CardContent
```

Comment:
![image](https://github.com/user-attachments/assets/e6ce8449-b9ae-4e22-af57-eb30644354f9)
Could you please revise the `CardContent` based on the Figma?